StuDash/urls.py

```diff
@@ -11,6 +12,8 @@
     path("", IndexPageView.as_view(), name="home"),
     path("bookmarks/", include("bookmarks.urls"), name="bookmarks"),
     path("grades/", include("grades.urls"), name="grades"),
+    path("messages", ViewMessages, name="Messages"),
```

Comment:
I see that the existing urls have trailing '/'
don't you want to be consistent? (or maybe you need it too)

Reply:
Figured I should, though since I wasn't sure what's the reason for that, I thought it better to leave it the way I understand and if needed, change to the '/' version in another PR with an explanation why I'm doing this.

Reply:
also, the capital letter on 'name' is not consistent

Reply:
But is it working like that?
I guess we will know only after the views will be merged into main

Reply:
Exactly what I thought. If it's not a complete mistake, I planned to merge it like this and if needed, improve it when I understand why.

Reply:
Consistent code within a project is important. First, add a code that looks like all the existing code,
If you think some other way is better, then you send another PR that fixes all the code (yours and others).
alternatively, you can fix their first and then add yours

Reply:
same for the '/'
consistent within the project

Reply:
Gotcha. Changed it and added an app-specific 'urls.py' accordingly, which can probably do more than it does now. I'll look into it further and adjust it accordingly (in a new dedicated PR).